lms/static/sass/views/_program-details.scss

```diff
@@ -303,6 +303,28 @@
     }
 
     .course-enroll-view {
+
+        .view-course-button {
+            background: palette(primary, dark);
+            border-color: palette(primary, dark);
+            height: 37px;
+            padding: 7px 18.5px 0 18.5px;
```

Comment:
It does not look the same to me on Chrome. Are you saying half pixels are not supported on some browsers or something? I did this padding because this gives the exact size of the button previously